api-docs/openapi-diff.js

```diff
@@ -0,0 +1,38 @@
+/* global process */
+/* eslint import/no-nodejs-modules:0 */
+/* eslint import/no-unresolved:0 */
+
+const fs = require('fs');
+const yaml = require('js-yaml');
+
+const jsonDiff = require('json-diff');
+
+const https = require('https');
+
+let sentry_api_schema_docs;
+https.get(
```

Comment:
is there a race condition on the https req and the file read, does this need to be in a promise? or should the other code live in the  `end` callback?